tests/flux-automation.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
this file should be added to CODEOWNERS, currently anyone can approve changes to it

Reply:
I've added the entire tests dir. I don't see why the other scripts there should not be owned as well.